### PR TITLE
Updated Docker client and compose versions in doc

### PR DESCRIPTION
**Pull request by stuclem**
Fixes https://github.com/vmware/vic-product/issues/2204.

@luwang-vmware what are the supported versions for Docker API and Docker Compose File? Thanks!

### CODE REVIEW

File: docs/user_doc/vic_app_dev/container_operations.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Supported Docker Commands
 
- vSphere Integrated Containers Engine 1.5 supports Docker client 1.13.0. The supported version of the Docker API is 1.25. 
+vSphere Integrated Containers 1.5.0 to 1.5.3 support Docker client 1.13.0. The supported version of the Docker API is 1.25. 
+
+vSphere Integrated Containers 1.5.4 and later versions support Docker client 18.09.6.
```
Comment (luwang-vmware) [inline docs/user_doc/vic_app_dev/container_operations.md]: Stuart, after using docker client 18.09 and docker-compose 1.24,  VIC CI hit a few issues which VIC engine need to update according. So in VIC 1.5.4, we can not announce to support it officially.

File: docs/user_doc/vic_app_dev/container_operations.md [hunk 2]
```diff
@@ -100,7 +102,9 @@ For more information about volume operations with vSphere Integrated Containers
 
 ## Docker Compose Commands <a id="compose"></a>
 
-vSphere Integrated Containers Engine 1.5 supports Docker Compose version 1.11.2.
+vSphere Integrated Containers 1.5.0 to 1.5.3 support Docker Compose version 1.11.2.
+
+vSphere Integrated Containers 1.5.4 and later versions support Docker Compose version 1.24.0.
```
Comment (luwang-vmware) [inline docs/user_doc/vic_app_dev/container_operations.md]: same as above.

File: docs/user_doc/vic_app_dev/container_operations.md (hunk 1 above)
Comment (stuclem) [reply docs/user_doc/vic_app_dev/container_operations.md]: OK, let's stick with the previous versions.

File: docs/user_doc/vic_app_dev/container_operations.md (hunk 2 above)
Comment (stuclem) [reply docs/user_doc/vic_app_dev/container_operations.md]: See above

**Comment by stuclem** [comment]
This is not valid for 1.5.4, and versions might differ by the time we release 1.5.5. Closing this PR.

**Pull request closed without merging**